src/prog.js

```diff
@@ -56,6 +56,7 @@ export default handler => {
 		.option('--name', 'Specify name exposed in UMD builds')
 		.option('--cwd', 'Use an alternative working directory', '.')
 		.option('--sourcemap', 'Generate source map', true)
+		.option('--worker-loader', 'Generate inline worker blobs', false)
```

Comment:
```suggestion
		.option('--workers', 'Bundle module workers - see https://git.io/J3oSF', false)
```

Reply:
for some reason eslint doesn't like it. I don't understand its error message:

```
59:11  error  Replace `'--workers',·'Bundle·module·workers·-·see·https://git.io/J3oSF',·false` with `⏎↹↹↹'--workers',⏎↹↹↹'Bundle·module·workers·-·see·h
ttps://git.io/J3oSF',⏎↹↹↹false,⏎↹↹`  prettier/prettier
```

Reply:
heh - it just wants the message to be broken across multiple lines (kinda silly in this case, but alas).

Reply:
OK, makes sense.